Tasks/Ananich/Task_4/Task_4.py

```diff
@@ -0,0 +1,28 @@
+user_symb = (input("Please enter max symbols in the string more than 35:\n"))
+if user_symb.isdigit() == True:
```

Comment:
I would suggest to do a cycle here and separate validation and actual solution. but this is not a huge mistake